app/src/main/java/com/sschuraytz/charcoaldrawing/EraseTool.java

```diff
@@ -0,0 +1,11 @@
+package com.sschuraytz.charcoaldrawing;
+
+import android.graphics.Color;
+
+
+public class EraseTool extends Tool{
+
+    public EraseTool() {
+        super(Color.WHITE);
```

Comment:
This is a good first step, but we probably want erase to "feel" a little bit different than charcoal as well.

Reply:
Okay. So I should potentially change the calculation to make the eraser a little smoother?

Reply:
> This is a good first step, but we probably want erase to "feel" a little bit different than charcoal as well.

Okay. So I should potentially change the calculation to make the eraser a little smoother?

Reply:
We can leave that for a different PR, but yes.